llvm/test/Transforms/HipStdPar/math-fixup.ll

```diff
@@ -0,0 +1,243 @@
+; CHECK-NEXT:    [[CALL21:%.*]] = call contract float @__hipstdpar_erf_f32(float noundef [[CALL19]]) #[[ATTR4]]
+; CHECK-NEXT:    [[CALL22:%.*]] = call contract double @__hipstdpar_erfc_f64(double noundef [[CALL20]]) #[[ATTR4]]
+; CHECK-NEXT:    [[CALL23:%.*]] = call contract float @__hipstdpar_erfc_f32(float noundef [[CALL21]]) #[[ATTR4]]
+; CHECK-NEXT:    [[CALL24:%.*]] = call contract double @__hipstdpar_tgamma_f64(double noundef [[CALL22]]) #[[ATTR4]]
+; CHECK-NEXT:    [[CALL25:%.*]] = call contract float @__hipstdpar_tgamma_f32(float noundef [[CALL23]]) #[[ATTR4]]
+; CHECK-NEXT:    [[CALL26:%.*]] = call contract double @__hipstdpar_lgamma_f64(double noundef [[CALL24]]) #[[ATTR3]]
+; CHECK-NEXT:    [[CALL27:%.*]] = call contract float @__hipstdpar_lgamma_f32(float noundef [[CALL25]]) #[[ATTR3]]
+; CHECK-NEXT:    ret void
+;
+entry:
+  %quo.addr = alloca i32, align 4
+  store i32 %quo, ptr %quo.addr, align 4
+  %0 = tail call contract double @llvm.fabs.f64(double %dbl)
+  %1 = tail call contract float @llvm.fabs.f32(float %flt)
+  %call = tail call contract double @remainder(double noundef %0, double noundef %0) #4
+  %call1 = tail call contract float @remainderf(float noundef %1, float noundef %1) #4
+  %call2 = call contract double @remquo(double noundef %call, double noundef %call, ptr noundef nonnull %quo.addr) #5
+  %call3 = call contract float @remquof(float noundef %call1, float noundef %call1, ptr noundef nonnull %quo.addr) #5
+  %2 = call contract double @llvm.fma.f64(double %call2, double %call2, double %call2)
+  %3 = call contract float @llvm.fma.f32(float %call3, float %call3, float %call3)
+  %call4 = call contract double @fdim(double noundef %2, double noundef %2) #4
+  %call5 = call contract float @fdimf(float noundef %3, float noundef %3) #4
+  %4 = call contract double @llvm.exp.f64(double %call4)
+  %5 = call contract float @llvm.exp.f32(float %call5)
+  %6 = call contract double @llvm.exp2.f64(double %4)
+  %7 = call contract float @llvm.exp2.f32(float %5)
+  %call6 = call contract double @expm1(double noundef %6) #4
+  %call7 = call contract float @expm1f(float noundef %7) #4
+  %8 = call contract double @llvm.log.f64(double %call6)
+  %9 = call contract float @llvm.log.f32(float %call7)
+  %10 = call contract double @llvm.log10.f64(double %8)
+  %11 = call contract float @llvm.log10.f32(float %9)
+  %12 = call contract double @llvm.log2.f64(double %10)
+  %13 = call contract float @llvm.log2.f32(float %11)
+  %call8 = call contract double @log1p(double noundef %12) #4
+  %call9 = call contract float @log1pf(float noundef %13) #4
+  %14 = call contract float @llvm.pow.f32(float %call9, float %call9)
+  %15 = call contract double @llvm.sqrt.f64(double %call8)
+  %16 = call contract float @llvm.sqrt.f32(float %14)
+  %call10 = call contract double @cbrt(double noundef %15) #4
+  %call11 = call contract float @cbrtf(float noundef %16) #4
```

Comment:
Remove the callsite attributes. Separate from this, probably should test that callsite function and parameter / return values are preserved (plus fpmath metadata)